src/dvclive/fastai.py

```diff
@@ -67,7 +67,8 @@ def after_epoch(self):
         # fast.ai calls after_epoch but we don't want to increase the step.
         if logged_metrics:
             if self.model_file:
-                self.learn.save(self.model_file, with_opt=self.with_opt)
+                file = self.learn.save(self.model_file, with_opt=self.with_opt)
+                self.live.log_artifact(str(file))
```

Comment:
@dberenbaum Opened this one to discuss before implementing all frameworks.

I think we should go ahead and deprecate `model_file` (https://github.com/iterative/dvclive/issues/499) but for now I just mad both approaches work (`model_file` and external callback).